syntax = "proto3";

package contracts.services.operations;

import "contracts/services/operations/operation.proto";

message CreateOperationRequest {
  OperationType type = 1;
  OperationStatus status = 2;
  float amount = 3;
  string card_id = 4;
  string category = 5;
  string created_at = 6;
  string account_id = 7;
}

message CreateOperationResponse {
  Operation operation = 1;
}